ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -125,8 +125,8 @@ private async Task<ImportResult> ValidateAndImportNotificationGroupAsync(Perform
                 var linkedNotificationId = notification.LegacyId;
                 _logger.LogInformation(context, requestId, $"Validating notification with Id={linkedNotificationId}");
 
-                var validationErrors = GetValidationErrors(notification);
-                if (validationErrors.Count() == 0)
+                var validationErrors = GetValidationErrors(notification).ToList();

```

Comment:
I assume this is in response to Rider telling us that there is possible multiple enumeration. I've actually picked up on this too in #392 , though in a slightly different way - I've changed the return type of `GetValidationErrors` to `ICollection` instead :)

Reply:
I will revert this back then

Reply:
oh don't worry about it, it'll get picked up in the merge! I just thought it interesting to point out